bootstrap/instance-terraform.tpl

```diff
@@ -0,0 +1,15 @@
+"Plugin": "instance-terraform",
+"Properties": {
+  "type": "aws_instance",
```

Comment:
this file should probably be named instance-terraform-aws.tpl or we'll have conflicts once we're ready to add other cloud providers with the same plugin.